src/arrayMethodJoin.js

```diff
@@ -4,8 +4,23 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let resultString = '';
```

Comment:
Mentors don't like a 'result' word. You should use “changedString” for example or something like this.

Reply:
thank you, I will change it)